engine/src/main/java/com/google/android/fhir/db/impl/dao/ResourceDao.kt

```diff
@@ -152,7 +172,18 @@ internal abstract class ResourceDao {
 
   @RawQuery abstract suspend fun countResources(query: SupportSQLiteQuery): Long
 
-  private suspend fun insertResource(resource: Resource): String {
+  suspend fun insertResourceLocal(resource: Resource, timeOfChange: Instant) =
```

Comment:
can we rename to this:
```suggestion
  suspend fun insertLocalResource(resource: Resource, timeOfChange: Instant) =
```

Reply:
Done